iOS Client/Models/MBPodcast.swift

```diff
@@ -7,8 +7,9 @@
 //
 
 import Foundation
+import UIKit
 
-struct MBPodcast: Codable, Detailable {
+struct MBPodcast: Codable {
```

Comment:
I like having two different models.

I think we should call this one `PodcastDTO` and `DisplayablePodcast` can be called `Podcast`.

I started this project by prefixing everything with `MB` b/c you had to use a custom prefix (usually 2 or 3 letters) in Objective-C world to avoid namespace collisions, but in swift that is no longer the case . . . 🤦‍♂️